Components/Avatar/avatar.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>UI-Avatar</title>
+    <link rel = "stylesheet" href = "/css/footer.css">
+    <link rel = "stylesheet" href = "/style.css">
+    <link rel = "stylesheet" href = "/css/navbar.css">
+    <link rel = "stylesheet" href = "/css/responsive.css">
+    <link rel = "icon" type = "image/x-icon" href = "/Images/favicon.png">
```

Comment:
You can put all of these CSS files in a folder. And import them in another index.css file

Reply:
Sure, I will make the changes.